api/src/test/java/org/openmrs/module/eptsreports/reporting/unit/calculation/EptsCalculationsTest.java

```diff
@@ -1,3 +1,42 @@
 package org.openmrs.module.eptsreports.reporting.unit.calculation;
 
-public class EptsCalculationsTest {}
+import static org.hamcrest.Matchers.is;
+
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.api.context.Context;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.patient.PatientCalculationService;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+public class EptsCalculationsTest extends BaseModuleContextSensitiveTest {
```

Comment:
@ningosi this is technically an integration test since it spins up a context. 

The reason I think we should keep these separate from unit tests is that they take longer to execute and we might want to eventually setup a separate test goal for testing these separately to our unit tests.

Reply:
Also I see this is infact for EptsCalculationsUtils so test class should be renamed to: EptsCalculationsTest.

Please rename and move to the integration test folder. Thanks :+1:

Reply:
Please also resolve codacy warning